.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: bionic
+git:
+  submodules: false
+before_install:
+  - git clone https://${TOKEN}:x-oauth-basic@github.com/meedan/configurator ./configurator
+before_script:
+  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+  - git submodule update --init --recursive
+  - sed -i 's/--abort-on-container-exit/-d/' bin/first-build.sh
+  - cd fetch
+  - git checkout develop
```

Comment:
I don't think we need to checkout `develop`, because the version to use is specified in the `.gitmodules` file. Or is there a good reason for it?